app/src/androidTest/java/org/systers/mentorship/RequestDetailActivityTestA.kt

```diff
@@ -0,0 +1,37 @@
+package org.systers.mentorship
+
+import android.content.Intent
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.rule.ActivityTestRule
+import org.junit.Rule
+import org.junit.Test
+import org.systers.mentorship.models.Relationship
+import org.systers.mentorship.utils.Constants
+import org.systers.mentorship.view.activities.MainActivity
+import org.systers.mentorship.view.activities.RequestDetailActivity
+
+class RequestDetailActivityTestA {
+
+    private val date: Float = System.currentTimeMillis().toFloat()
+    private val relationship = Relationship(
+            0, 0, true,
+            Relationship.RelationUserResponse(0, "test_user_1"),
+            Relationship.RelationUserResponse(1, "test_user_2"),
+            date, 0.0F, 0.0F, date, 1, "test"
+    )
+
+    @get:Rule
+    var activityRule = ActivityTestRule(MainActivity::class.java)
+
```

Comment:
Can you please add a small comment explaining what this function is doing?

Reply:
Sure, Thanks for the review, I've added comments for each statement!